### PR TITLE
Migrate from Travis CI to GitHub Action

### LINKED ISSUE #1221: Move from Travis CI to Github Actions
Our CI settings are a bit outdated - they just `npm install` - we should move them to Github actions so the whole team can have full access to the whole process. It should run:
* `npm install`
* `npm start`
* `npm run docs`

### PR DESCRIPTION
**Pull request by Dinesh101041**
Migrate from Travis CI to GitHub Action

1. node.yaml - Action for javascript
2. css.yaml - Action for css 
3. lint.yaml - Action for lints

Deleted  Travis.yml workflow

Fix #1221

### CODE REVIEW

**Comment by eltonmesquita** [comment]
Thanks for the PR @Dinesh101041. I've been in a short time schedule lately, but I should be able to review your lovey PR in the next few days. Sorry for taking so much time.

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        uses: actions/checkout@v2
+        with:
+          # Full git history is needed to get a proper list of changed files within `super-linter`
+          fetch-depth: 0
+
+      ################################
+      # Run Linter against code base #
+      ################################
+      - name: Lint Code Base
+        uses: github/super-linter@v3
```
Comment (eltonmesquita) [inline .github/workflows/lint.yaml:51]: I didn't know this linter but I had a quick look and we might be better with a simple eslinter setup. I can add eslinter and prettier so you can update this to a simple npm package script. What do you think?

File: .github/workflows/node.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install
+    - run: npm start
+    - run: npm run docs
+    - run: npm run build --if-present
```
Comment (eltonmesquita) [inline .github/workflows/node.yaml:27]: Will this add the compiled files to the current PR? Else, what's the end result?

**Review by eltonmesquita — CHANGES_REQUESTED** [review]
I'm so sorry for taking so long to review, but I'm back in business and should give feedback faster from now on.

Thanks for the PR. There are a few questions as you can see.

File: .github/workflows/node.yaml (hunk 2 above)
Comment (Dinesh101041) [reply .github/workflows/node.yaml:27]: i am not well in github actions i done this on reading github node workflow doc 
here is link of two docs [Github node workflow](https://docs.github.com/en/actions/guides/building-and-testing-nodejs) 
i will delete the linter and css workflow we can add the eslinter setup

**Comment by WarenGonzaga** [comment]
I think this PR has some conflicts... can you update this @Dinesh101041?

File: .github/workflows/node.yaml (hunk 2 above)
Comment (WarenGonzaga) [reply .github/workflows/node.yaml:27]: Any updates on this?

Comment (WarenGonzaga) [reply .github/workflows/node.yaml:27]: > > Any updates on this?
> 
> can you please elaborate the conflict and i will delete this pull and make a new pull with all update

@eltonmesquita can you please elaborate on it?

**Comment by WarenGonzaga** [comment]
Hi @Dinesh101041 any update on this? cc: @eltonmesquita just checking things here.

**Comment by Dinesh101041** [comment]
> Hi @Dinesh101041 any update on this? cc: @eltonmesquita just checking things here.

not able to work after that due to other works

**Comment by eltonmesquita** [comment]
I'll close this one as it's outdated by now. I'll take some time and do it myself as it's a task that involves a few other ones.

**Pull request closed without merging**